lib/wiki_edit_mappings.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# Class returns template mappings for WikiEdu and P&E Dashboards, respectively
+class WikiEditMappings
```

Comment:
We can see how far we get with hard-coding the template names; I think it's an okay way to do things for now.

Alternatives include putting the mapping into a config file or locale file, or putting it into the database (which would let us make a simple frontend for setting the templates for a particular wiki).